spec/models/category_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Category do
+  let!(:current_user) { create(:user) }
+
+  context 'with valid params' do
+    it 'is valid' do
+      category = described_class.new(name: 'Category', user: current_user)
+
+      expect(category).to be_valid
+    end
+
+    it 'generates slug' do
+      category = described_class.create(name: 'Category name', user: current_user)
+
+      expect(category.slug).to match('category-name')
+    end
+  end
+
+  describe 'model validations' do
```

Comment:
Lá em cima tem um context "with valid params" então acho que isso aqui poderia seria um:
```suggestion
  context 'with invalid params' do
```

Daí esse último teste `it 'is valid when another user has category with same name'` vc move pro context de cima